modules/backend/classes/BackendController.php

```diff
@@ -203,4 +258,17 @@ protected function parseAction($actionName)
 
         return $actionName;
     }
-}
\ No newline at end of file
+
+    /**
+     * Determine if the given options exclude a particular method.
+     *
+     * @param  string  $method
+     * @param  array  $options
+     * @return bool
+     */
+    protected static function methodExcludedByOptions($method, array $options)
+    {
+        return (isset($options['only']) && ! in_array($method, (array) $options['only'])) ||
```

Comment:
@bennothommo could you use only one-way: space after `!` or none at all 😄  (68,136: `!is_null`, 62: `!empty`, 270: `! in_array`)

Reply:
@w20k In my defence, I copied Laravel's homework :P
https://github.com/laravel/framework/blob/aa36e209480de3bb36036a959fc3b114a21e7ebf/src/Illuminate/Routing/ControllerDispatcher.php#L76

Reply:
Thanks, @bennothommo!